high_card.rb

```diff
@@ -3,3 +3,60 @@
 suits = [ "hearts", "spades", "clubs", "diamonds" ]
 
 players = []
+
+deck = Array.new
+
+# create deck from ranks and suits
+ranks.each do |rank|
+  suits.each do |suit|
+    deck.push([rank, suit])
+  end
+end
+
+# verify deck is complete
+puts deck.inspect
+puts deck.length
+
+# unfinished: create array of arrays of ranks with their numeric value (Aces high)
+card_value = Hash.new
+
+# get either player names or 'play' command
+def start_game(players)
+  puts "#{players.length} players so far. Enter a player name, or type 'play':"
+  user_input = gets.chomp
+  return user_input
+end
+
+# does this call the above?
+user_input = start_game(players)
```

Comment:
Yes, and passes it the players you have at that time, which looks like the empty array from line 5.